yarn.lock

```diff
@@ -2134,7 +2136,7 @@
 "@wora/apollo-cache@file:packages/apollo-cache":
   version "2.2.0"
   dependencies:
-    "@wora/cache-persist" "file:packages/cache-persist"
+    "@wora/cache-persist" "file:../../Library/Caches/Yarn/v6/npm-@wora-apollo-cache-2.2.0-1e7edac8-41ff-46b5-a71d-4fe16e9444c1-1611579151904/node_modules/@wora/cache-persist"
```

Comment:
These look incorrect but is what yarn is throwing out at me. 🤔

Reply:
That is a known issue: https://github.com/yarnpkg/yarn/issues/4388 😅